Sources/SwiftCompilerPlugin/CompilerPlugin.swift

```diff
@@ -12,12 +12,22 @@
 // NOTE: This basic plugin mechanism is mostly copied from
 // https://github.com/apple/swift-package-manager/blob/main/Sources/PackagePlugin/Plugin.swift
 
-@_implementationOnly import Foundation
-@_implementationOnly import SwiftCompilerPluginMessageHandling
 import SwiftSyntaxMacros
 
+#if swift(>=5.11)
```

Comment:
How about
```suggestion
#if hasFeature(AccessLevelOnImport)
```

Reply:
It turned out `hasFeature(AccessLevelOnImport)` doesn't work.

Reply:
Discussed offline that `AccessLevelOnImport` is an experimental feature that’s always enabled, thus the `hasFeature` check doesn’t work. So we just check for Swift 5.11.